chanbackup/recover.go

```diff
@@ -57,9 +59,10 @@ func Recover(backups []Single, restorer ChannelRestorer,
 			continue
 		}
 		if err != nil {
-			return err
+			return 0, err
```

Comment:
Q: Should we return `numRestored` here instead since technically this number of channels have been successfully restored?